.gitignore

```diff
@@ -1,4 +1,5 @@
 /.idea/
+/.php_cs.cache
```

Comment:
Should be put in your global `.gitignore` ideally

Reply:
I would prefer to keep it in the project, as it is generated IN the repo and should not accidently be commited!

Reply:
I agree with @OskarStark, i give it back

Reply:
Yes and no ; yes, it is generated in the project... but no, because it's the result of a *tool* used on the project.

Otherwise, we'd need to do the same with the phpstorm config, ... and so on. And we'd never get out.

Reply:
@Taluu I do not follow you, we already ignore the phpstorm configuration, it is an external tool but that modify the project

Reply:
agreed with @maxhelias : given.we provide a `.php_cs.dist` file, this is related to the bundle. like `vendor` is related to project even if composer is an external tool.

But `.idea` is definitively related to the developer and should be removed